src/CommunityToolkit.Maui.Camera/CameraManager.windows.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Runtime.Versioning;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Extensions;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.UI.Xaml.Controls;
+using Windows.Media.Capture;
+using Windows.Media.Capture.Frames;
+using Windows.Media.Core;
+using Windows.Media.MediaProperties;
+
+namespace CommunityToolkit.Maui.Core;
+
+[SupportedOSPlatform("windows10.0.10240.0")]
+partial class CameraManager
+{
+	MediaPlayerElement? mediaElement;
+	MediaCapture? mediaCapture;
+	MediaFrameSource? frameSource;
+
+	public MediaPlayerElement CreatePlatformView()
+	{
+		mediaElement = new MediaPlayerElement();
+		return mediaElement;
+	}
+
+	public void Dispose()
+	{
+		Dispose(true);
+		GC.SuppressFinalize(this);
```

Comment:
No need for this call, as mentioned before
```suggestion
```

Reply:
It's cool to keep it. It'll save us a bug in the future if a PR adds a finalizer and/or a dev subclasses this and adds a finalizer.